tests/build_/test_plugins_configuration.py

```diff
@@ -104,7 +104,7 @@ def assert_koji_upload_plugin(self, plugins):
 
         assert get_plugin(plugins, phase, name)
         plugin_args = plugin_value_get(plugins, phase, name, 'args')
-        assert plugin_args.get('build_json_dir')
+        assert plugin_args.get('koji_upload_dir')
```

Comment:
Maybe verify it's set to the expected value?

* Duplicating this comment because the previous one got hidden along the way...

Reply:
Fixed.